robottelo/cli/base.py

```diff
@@ -73,25 +73,30 @@ class Base:
 
     @classmethod
     def _handle_response(cls, response, ignore_stderr=None):
-        """Verify ``return_code`` of the CLI command.
+        """Verify ``status`` of the CLI command.
 
         Check for a non-zero return code or any stderr contents.
 
-        :param response: a ``SSHCommandResult`` object, returned by
-            :mod:`robottelo.ssh.command`.
+        :param response: a result object, returned by :mod:`robottelo.ssh.command`.
         :param ignore_stderr: indicates whether to throw a warning in logs if
             ``stderr`` is not empty.
         :returns: contents of ``stdout``.
         :raises robottelo.cli.base.CLIReturnCodeError: If return code is
             different from zero.
         """
-        if response.return_code != 0:
+        if isinstance(response.stderr, tuple):
+            # the current contents of response.stderr is a tuple with the following properties
+            # (<len(message)>,<message>).
+            # This behavior could (and maybe should) change in the near future.
+            # In the meantime, we don't need it here so we just use the message itself
+            response.stderr = response.stderr[1]
+        if response.status != 0:
             full_msg = (
                 f'Command "{cls.command_base} {cls.command_sub}" '
-                f'finished with return_code {response.return_code}\n'
+                f'finished with status {response.status}\n'
```

Comment:
This move/refactor includes renaming everything to `status` instead of `return_code`, as that's the naming that ssh2-python uses.

Reply:
@mshriver Yup! I was more curious to update it from Robottelo code base even though `ssh2-python` lib but since you shared the standard exit_status doc on automation channel, so I am good with it.